packages/prisma/migrations/20230524105015_added_newbooker_feature_flag/migration.sql

```diff
@@ -3,7 +3,7 @@ INSERT INTO
 VALUES
   (
     'booker-layouts',
-    false,
+    true,
```

Comment:
We cannot and should not modify already applied migrations.

```suggestion
    false,
```